_docs/faq.markdown

```diff
@@ -1,30 +1,36 @@
 ---
 title: FAQ
-order: 999999
+order: 3
 ---
 
 {% include h title="Where is the Rust SDK?" tag="sdk" %}
 
-We don't use IntelliJ IDEA's SDK concept to manage Rust versions because it is
-specific to IDEA and is not present, for example, in CLion. The path to Cargo is
-configured per project through `Settings > Languages & Frameworks > Rust`.
+We don't use the SDK concept to manage Rust versions because it is
+specific for IntelliJ IDEA and is not present in other IDEs such as CLion. The path to Cargo is
+configured per project in `Settings / Preferences | Languages & Frameworks | Rust`.
 
-{% include h title="Does IntelliJ Rust have a debugger?" tag="debugger" %}
 
-There is preliminary debugger support for CLion, see [this issue][debugger] for
-details.
+{% include h title="What are the compatible IDEs? Is there any difference?" tag="compatible-ides" %}
+The plugin works with all IntelliJ-based IDEs of the version 2019.2 and newer. 
```

Comment:
Let's avoid a concrete version here because it will require updating every four months. And I'm sure that we will forget to update it.
For example, the latest plugin update is the latest version that supports 2019.2